setup/git/020_PuppetUserAndGroup.rb

```diff
@@ -1,10 +1,7 @@
 test_name 'Puppet User and Group' do
   hosts.each do |host|
-
     step "ensure puppet user and group added to all nodes because this is what the packages do" do
       on host, puppet("resource user puppet ensure=present")
-      on host, puppet("resource group puppet ensure=present")
```

Comment:
this was causing issues to have with osx tests. Once I removed it, osx tests started passing and no other platforms were impacted